### ISSUE TITLE
Glitches In MineKhan

### ISSUE THREAD

**Issue by SnehasishCodez** [Willard21/MineKhan#32]
Here is a very astonishing glitch I found in minekhan it's a spin-off
https://www.khanacademy.org/computer-programming/minecraft-but-broken/4914589619929088

You may notice that your x, y coordinates are VERY VERY high, meaning that we hundred of thousands of blocks away from spawn which makes the game TOO glitchy

If you are a Minecraft nerd then you might know about the far lands which are basically on the topic of how infinite can you make procedurally generated terrain if you do not know what far lands is here is a summary :

The Far Lands is a terrain bug that appears upon the overflowing of a noise generator, most notably the low and high noise overflow 12,550,821 blocks from the origin of the Minecraft world. There are other parts of Far Lands called Farther Lands and Corner Far Lands.

**Comment by Hacker1254** [comment]
Bruh i was bout to show this
![image](https://user-images.githubusercontent.com/69991448/117979488-f5ca9b80-b300-11eb-96cf-15e393cc33b9.png)

i even had a video and all

**Comment by Willard21** [comment]
Lol I've known about this since before I released the game. But noise overflow has nothing to do with it, unlike the real Minecraft farlands. This is caused by too low float precision in the vertex shaders. It's the highest precision available though. It's fixable, but would cause a universal performance decrease. Since people basically have to hack the game to get out there, it's not worth it.

**Comment by Hacker1254** [comment]
💯

**Comment by SnehasishCodez** [comment]
@Willard , so you are saying it's not possible to get to the glitch on foot so it's only possible by hacking or modyfying.

**Comment by Hacker1254** [comment]
Yea

**Comment by SnehasishCodez** [comment]
ok

**Comment by Hacker1254** [comment]
oh my god
![image](https://user-images.githubusercontent.com/69991448/118288599-93a99c00-b4a2-11eb-8bd8-76047d0c47c5.png)
i was just trying to make a switch

**Comment by SnehasishCodez** [comment]
> oh my god
> ![image](https://user-images.githubusercontent.com/69991448/118288599-93a99c00-b4a2-11eb-8bd8-76047d0c47c5.png)
> i was just trying to make a switch

Can you make a project and try to change the x and y to astronomically levels and then make a spin off and send here ?

**Comment by sauron65** [comment]
> Yea, it will take many years

I don't think you even could make it that far no matter how much time you have because the game will run out of memory.

**Comment by Willard21** [comment]
Lol yeah, chunks stay loaded forever after they're generated. So would definitely run out of memory.

**Comment by sauron65** [comment]
@SnehasishCodez
> @willard , so you are saying it's not possible to get to the glitch on foot so it's only possible by hacking or modyfying.

I actually did get the glitch on foot. I put something this in the url box:
```js
javascript:player.x = 1000000
```

**Comment by Willard21** [comment]
Well the obvious solution to this is to remove global access to the `player` object then, since this is clearly something everybody does during normal gameplay.
